lib/node-token-validation/src/network/HttpClient.ts

```diff
@@ -50,12 +45,55 @@ export class HttpClient implements INetworkModule {
             validateStatus: () => true
         };
 
-        const response = await axios(request);
+        return this.sendAxiosRequest(request, url);
+    }
+
+    /**
+     * Helper function to handle network retries for Axios requests.
+     * Function will retry axios request twice when response is returned with HTTP status code indicating request timeout or service unavailable. 
+     * 
+     * @param {AxiosRequestConfig} request Axios request
+     * @param {string} url URL for request
+     * @returns 
+     */
+    private async sendAxiosRequest<T>(request: AxiosRequestConfig, url: string): Promise<NetworkResponse<T>> {
```

Comment:
This is really neat. Are there any plans to add this functionality to MSAL-Node?